CHANGES.rst

```diff
@@ -15,6 +15,8 @@ assign_wcs
 associations
 ------------
 
+- Update to prevent target_acquisitions from processing in the spec3 pipeline [#3777]
```

Comment:
Minor nit, but these changes are really in `datamodels` and `calwebb_spec3`, as opposed to the `associations` infrastructure or rules, so I'd categorize them in a different section of the change log.

Reply:
Moved this comment to the datamodels section.